python/mxnet/_numpy_op_doc.py

```diff
@@ -21,7 +21,8 @@
 
 
 def _np_reshape(a, newshape, order='C'):
```

Comment:
Please add parameter ``out`` for ``np.reshape`` . It is the fourth argument.
```
>>> a = np.arange(6)
>>> a
array([0., 1., 2., 3., 4., 5.])
>>> b = np.zeros((3, 2))
>>> np.reshape(a, (3, 2), out=b)
array([[0., 1.],
       [2., 3.],
       [4., 5.]])
>>> b
array([[0., 1.],
       [2., 3.],
       [4., 5.]])
```

Reply:
@gyshi

Reply:
resolve